### PR TITLE
Rdm 9052

### PR DESCRIPTION
**Pull request by henrykcgi**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: liquibase.gradle [hunk 1]
```diff
@@ -3,7 +3,7 @@ configurations {
 }
```
Comment (mario-paniccia) [inline liquibase.gradle:4]: in the previous PR we introduced db.changelog-RDM-9052.xml
Are we able to get rid of it now after the update to the newest liquibase version or we still need that?

Comment (henrykcgi) [reply liquibase.gradle:4]: We got rid of it.

**Review by mario-paniccia — APPROVED** [review]
👍

File: src/main/resources/db/changelog/db.changelog-master.xml [hunk 2]
```diff
@@ -39,6 +39,5 @@
     <include file="db/changelog/db.changelog-RDM-8463.xml"/>
     <include file="db/changelog/db.changelog-RDM-9287.xml"/>
     <include file="db/changelog/db.changelog-RDM-9505.xml"/>
-    <include file="db/changelog/db.changelog-RDM-9052.xml"/>
```
Comment (mario-paniccia) [inline src/main/resources/db/changelog/db.changelog-master.xml:42]: just removing the file won't work on environments were the changelog has already been applied, for example Demo. For that case you'd need to explicitly introduce a new changelog that undo the changed introduced by `changelog-RDM-9052`. And which runs only if `changelog-RDM-9052` has run already.

**Review by noronhaa — APPROVED** [review]

**Pull request merged**